### PR TITLE
Add outputs for all workflows

### PR DESCRIPTION
**Pull request by gabriel-farache**
Closes https://issues.redhat.com/browse/FLPATH-1795

Add outputs for all workflows to avoid empty result in UI

### CODE REVIEW

**Comment by ElaiShalevRH** [comment]
looks good, but those aren't all the workflows, right?
The ones you've omitted  didn't need any output schema changes to this format?

**Comment by gabriel-farache** [comment]
@ElaiShalevRH the assessments workflows already have the output schema set (MTV plan and MTA), or did I miss infra workflow?

File: modify-vm-resources/modify-vm-resources.sw.yaml [hunk 1]
```diff
@@ -42,6 +44,37 @@ functions:
...
+              "value": $SECRET.cluster_console_url + "/k8s/ns/" + .vm_namespace + "/kubevirt.io~v1~VirtualMachine/" + .vm_name + "/console/standalone",
+              "format":"link"
+          }
+        ]
+      }'
+  - name: errorVMNotRunningResult
+    type: expression
+    operation: '{
+        "completedWith":"error",
+        "message": "VM" + .vm_name + " in namespace " + .vm_namespace + " not ready after the update after checking" +.vmStatusRunningRetries+" times: " + .vm.status
```
Comment (masayag) [inline modify-vm-resources/modify-vm-resources.sw.yaml:64]: to match the spacing  pls s/`+.vmStatusRunningRetries+"`/`+ .vmStatusRunningRetries +"`

File: mtv-migration/schemas/workflow-output-schema.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "WorkflowResult",
+    "description": "Schema of workflow output",
+    "type": "object",
+    "properties": {
+        "result": {
+            "$ref": "../shared/schemas/workflow-result-schema.json",
```
Comment (masayag) [inline mtv-migration/schemas/workflow-output-schema.json:8]: is this the correct path? I don't recall where we create the `shared` folder

Comment (gabriel-farache) [reply mtv-migration/schemas/workflow-output-schema.json:8]: https://github.com/parodos-dev/serverless-workflows/blob/main/Makefile#L107 is making sure the shared folder is copied next to the workflow https://github.com/parodos-dev/serverless-workflows/blob/main/shared/schemas/workflow-result-schema.json

File: modify-vm-resources/modify-vm-resources.sw.yaml (hunk 1 above)
Comment (gabriel-farache) [reply modify-vm-resources/modify-vm-resources.sw.yaml]: done

**Review by masayag — APPROVED** [review]

**Pull request merged**